### PR TITLE
Small changes in `perplexity.rst`to make the notebook executable on google collaboratory

### PR DESCRIPTION
**Pull request by SaulLu**
# What does this PR do?

Small changes in `perplexity.rst`to make the notebook executable on google collaboratory.

Replace this [PR](https://github.com/huggingface/notebooks/pull/85) in the `notebooks` repository.

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/master/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/master/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/master/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. @sgugger, I would love to have your thoughs on this PR.

### CODE REVIEW

**Comment by SaulLu** [comment]
I take the opportunity to point out that a change was already made in this file and not in the corresponding file into `notebooks` repository: `from nlp import load_dataset` -> `from datasets import load_dataset`. 

Whereas when we press the `open in Colab` button in the [documentation](https://huggingface.co/transformers/perplexity.html), we get the notebook stored into the `notebooks` repository.

File: docs/source/perplexity.rst [hunk 1]
```diff
@@ -109,6 +109,8 @@ for context when calculating the conditional likelihood of any one token (provid
 available to condition on).
 
 .. code-block:: python
+    import torch
```
Comment (sgugger) [inline docs/source/perplexity.rst:112]:
```suggestion

    import torch
```
You need to start new code blocks with an empty line.

**Review by sgugger — APPROVED** [review]
Thanks for fixing! Will upload the updated notebooks after this is merged.

**Comment by SaulLu** [comment]
> Thanks for fixing! Will upload the updated notebooks after this is merged.

With pleasure it was really a small change! To understand the workflow, what we have to do is edit the files in the `docs/source` folder, generate automatically the notebooks from these files and then upload manually the modified notebooks in the `notebooks` repository? :blush:

**Comment by sgugger** [comment]
Yes, the exact workflow is to have an up to do date repo of transformers in the same folder has the notebooks repo, then in the notebooks repo, make sure we are up to date with master, run `make doc-notebooks`, commit and push the changes.

**Pull request merged**